src/app/pages/Swap/form/Form.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { memo } from 'react';
+
+import { TempleChainKind } from 'temple/types';
+
+import { EvmSwapForm } from './EvmSwapForm';
+import { SwapReviewData } from './interfaces';
+import { TezosSwapForm } from './TezosSwapForm';
+
+interface SwapFormProps {
+  chainKind?: string | null;
+  chainId?: string | number | null;
+  activeField: 'from' | 'to';
```

Comment:
`type SwapFieldName = 'from' | 'to'; ` in form/interfaces.ts, and reuse where appropriate